lib/rucio/daemons/reaper/reaper2.py

```diff
@@ -240,6 +240,13 @@ def get_rses_to_hostname_mapping():
 
     result = REGION.get('rse_hostname_mapping')
     if result is NO_VALUE:
+        semaphore = REGION.get('semaphore')
+        if semaphore is NO_VALUE:
```

Comment:
The semaphore should be removed. I don't think it has the wished effect and will not do much.